package-lock.json

```diff
@@ -1336,7 +1336,7 @@
           "integrity": "sha512-VT0ZI6kZRdTh8YyJw3SMbYm/u+NqfsAxEpWO0Pf9sq8/e94WxxOpPKx9FR1FlyCtOVDNOQ+8ntlqFxiRc+r5qA==",
           "dev": true,
           "requires": {
-            "color-convert": "1.9.1"
+            "color-convert": "^1.9.0"
```

Comment:
I'm not too sure why running `npm install` even on `master` leads to a diff in `package-lock.json` 😕

Reply:
Either someone before you wasn't using the latest version, or you need to update. 6.4.1 is the latest, I think.